content/sensu-core/2.0/reference/mutators.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Mutators"
+description: "The mutators reference guide."
+weight: 1
+version: "2.0"
+platformContent: False 
+menu:
+  sensu-core-2.0:
+    parent: reference
+---
+
+# Mutators
```

Comment:
I don't think that header is required, otherwise it produces the following:

![screen shot 2018-03-14 at 10 28 22 am](https://user-images.githubusercontent.com/4105580/37408421-719d8510-2772-11e8-8eee-3e17d9f9a5dc.png)

Reply:
🤦‍♀️